ui/pages/settings/contact-list-tab/contact-list-tab.component.js

```diff
@@ -105,35 +104,25 @@ export default class ContactListTab extends Component {
     const { history, viewingContact, editingContact } = this.props;
 
     return (
-      <div className="address-book-add-button">
-        <Button
-          className={classnames({
-            'address-book-add-button__button': true,
-            'address-book-add-button__button--hidden':
-              viewingContact || editingContact,
-          })}
-          type="secondary"
-          onClick={() => {
-            history.push(CONTACT_ADD_ROUTE);
-          }}
-        >
-          {this.context.t('addContact')}
-        </Button>
-      </div>
+      <ButtonPrimary
+        className={classnames({
+          'address-book-add-button__button': true,
```

Comment:
If a classname should always be added (in this case `true`), you should do:

```
className={classnames('address-book-add-button__button', { ....
```

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/19465/commits/0348f3a5a0a5b4c325fcc52a02a11b3d2c645789#diff-cdd29e9c6dd899bd11506911a21f0ad3d55b5a11dd0389357d05440168547e90R107-R108